WalletWasabi/Blockchain/TransactionOutputs/SmartCoin.cs

```diff
@@ -166,9 +173,9 @@ public bool RefreshAndGetIsBanned()
 	public bool IsSpent() => SpenderTransaction is not null;
 
 	/// <summary>
-	/// IsUnspent() AND !SpentAccordingToBackend AND !CoinJoinInProgress
+	/// IsUnspent() AND !SpentAccordingToBackend AND !CoinJoinInProgress AND !IsExcludedFromCoinJoin
 	/// </summary>
-	public bool IsAvailable() => SpenderTransaction is null && !SpentAccordingToBackend && !CoinJoinInProgress;
+	public bool IsAvailable() => SpenderTransaction is null && !SpentAccordingToBackend && !CoinJoinInProgress && !IsExcludedFromCoinJoin;
```

Comment:
This function is messy. I think it should be removed. It is not clear. Available for what? CoinJoin? Send?

Do not touch it and add the condition here instead:
https://github.com/molnard/WalletWasabi/blob/master/WalletWasabi/WabiSabi/Client/CoinJoinManager.cs#L503